packages/bora-vanilla/src/js/utils.js

```diff
@@ -0,0 +1,28 @@
+import { ERROR_CODE } from './constants';
+
+export const formatter = new Intl.NumberFormat();
+
+/** 문자열에서 숫자만 추출합니다. */
+export const extractDigitsOnly = (value) => {
+  return value.replace(/[^\d]/g, '');
```

Comment:
minor: 다른 분께도 알려드린 적 있는데 `[^\d]`는 `\D`로 간소화 할 수 있습니다.
```suggestion
  return value.replace(/\D/g, '');
```